metron-interface/metron-config/package.json

```diff
@@ -33,6 +28,11 @@
     "@angular/platform-browser": "2.0.0",
     "@angular/platform-browser-dynamic": "2.0.0",
     "@angular/router": "3.0.0",
+    "@types/ace": "0.0.32",
```

Comment:
Were there any real changes made to this file?  It looks like some dependencies were simply rearranged.  I do see the addition of `"main": "index.js"`.  What is the purpose of that?

Reply:
No there were no changes other than rearrange of the dependencies.
About the main attribute, in our case it will do nothing. It defines the entry point of the application if we would SystemJS or requireJS. It is the default setting for yarn.

Reply:
What was the reason for rearranging the dependencies?  It's not a big deal but it's distracting and makes reviewing harder.

Reply:
Yes. I was not enough carefully. Yarn did the rearrangement, but I can move the dependencies back to their original place if you would like to.

Reply:
No I was just curious.  If Yarn did this automatically we might as well leave it that way.  No need to revert.